### DISCUSSION TITLE
get rid of unenroll

### DISCUSSION THREAD

**Discussion by velzie** [3kh0/ext-remover#164] [Fixes JOIN TITANIUM NETWORK TO SHARE PLEASE]
you know k12sysadmin reloads this page every 5 seconds right

**Comment by Sincereham222** [comment]
> fuck you

Woah woah don't say that naughty fuck word

**Comment by ShyMaster202** [comment]
agreed

**Comment by 3kh0** [comment]
yeah I do know

**Comment by deryilz** [comment]
As a system administrator, I can confirm that I have already messaged Google about this. This is a disgusting and borderline illegal abuse of our generosity. If I find who you are, I guarantee that I will report you to your local police departments. Expect an all-version fix coming for this bug in around two days. Can't say much more than that. I have messaged the GitHub team to take this repository and account down, and am messaging Discord's team about "Titanium Network" as well. Prepare yourselves.

**Reply by vasileferentari** [reply]
did i ask LMAO

**Reply by velzie** [reply]
L + ratio

**Reply by vasileferentari** [reply]
My name is Walter Hartwell White. I live at 308 Negra Arroyo Lane Albuquerque New Mexico 87104. This is my confession; I hacked my school Chromebook to allow myself to install a different operating system.

**Reply by ShyMaster202** [reply]
My name is Walter Hartwell White. I live at 308 Negra Arroyo Lane Albuquerque New Mexico 87104. This is my confession; I hacked my school Chromebook to allow myself to install a different operating system.

**Reply by deryilz** [reply]
GUYS ITS A JOKE :sob:

**Reply by ShyMaster202** [reply]
As a system administrator, I can confirm that I have already messaged Google about this. This is a disgusting and borderline illegal abuse of our generosity. If I find who you are, I guarantee that I will report you to your local police departments. Expect an all-version fix coming for this bug in around two days. Can't say much more than that. I have messaged the GitHub team to take this repository and account down, and am messaging Discord's team about "Titanium Network" as well. Prepare yourselves.

**Reply by vasileferentari** [reply]
nah bruh u real

**Reply by deryilz** [reply]
slight trolling

**Reply by vasileferentari** [reply]
u is real man

**Reply by vasileferentari** [reply]
what u discord??

**Reply by deryilz** [reply]
learn grammar my guy

**Reply by vasileferentari** [reply]
what U DISCORD???

**Reply by deryilz** [reply]
learn grammar

**Reply by vasileferentari** [reply]
ik grammar so what u discord

**Reply by synapt0** [reply]
> As a system administrator, I can confirm that I have already messaged Google about this. This is a disgusting and borderline illegal abuse of our generosity. If I find who you are, I guarantee that I will report you to your local police departments. Expect an all-version fix coming for this bug in around two days. Can't say much more than that. I have messaged the GitHub team to take this repository and account down, and am messaging Discord's team about "Titanium Network" as well. Prepare yourselves.

nah cuz I legit thought u were being frfr

**Reply by 3kh0** [reply]
You know you are doing something right when the sys admins start coming after you, flooding your Dms with TAKE IT DOWNNNN

**Reply by Unciaur** [reply]
And then discord is like “Do not harass others or organize, promote, or participate in harassment.” No, Google probably doesn’t care and No, Discord probably doesn’t care. Have fun in the 5 month waitline.

**Reply by ShyMaster202** [reply]
> You know you are doing something right when the sys admins start coming after you, flooding your Dms with TAKE IT DOWNNNN

its a joke 💀

**Reply by 3kh0** [reply]
@ShyMaster202 This is not the only one

**Comment by vasileferentari** [comment]
My name is Walter Hartwell White. I live at 308 Negra Arroyo Lane Albuquerque New Mexico 87104. This is my confession; I hacked my school Chromebook to allow myself to install a different operating system.

**Reply by ShyMaster202** [reply]
My name is Walter Hartwell White. I live at 308 Negra Arroyo Lane Albuquerque New Mexico 87104. This is my confession; I hacked my school Chromebook to allow myself to install a different operating system.

**Reply by deryilz** [reply]
My name is Walter Hartwell White. I live at 308 Negra Arroyo Lane Albuquerque New Mexico 87104. This is my confession; I hacked my school Chromebook to allow myself to install a different operating system.

**Reply by 3kh0** [reply]
My name is Walter Hartwell White. I live at 308 Negra Arroyo Lane Albuquerque New Mexico 87104. This is my confession; I hacked my school Chromebook to allow myself to install a different operating system.

**Reply by Yeetusmans2048** [reply]
My name is Walter Hartwell White. I live at 308 Negra Arroyo Lane Albuquerque New Mexico 87104. This is my confession; I hacked my school Chromebook to allow myself to install a different operating system.

**Comment by velzie** [comment]
you know the link is dead now right? i changed how the scripts are pulled

**Reply by synapt0** [reply]
My name is Walter Hartwell White. I live at 308 Negra Arroyo Lane Albuquerque New Mexico 87104. This is my confession; I hacked my school Chromebook to allow myself to install a different operating system.

**Reply by 3kh0** [reply]
@CoolElectronics I am fully aware of that
